src/Popover/Popover.js

```diff
@@ -371,7 +376,50 @@ export default class Popover extends React.Component {
     );
   };
 
-  getVisiblePopover = () => this.popoverTemplate(this.computePopoverStyle());
+  getVisiblePopover = () => {
+    const { popover } = this.state;
+    const popoverProps = this.getPopoverProps();
+
+    if (popoverProps.auto) {
+      const positions = uniq([popoverProps.position, 'top', 'bottom', 'left', 'right']);
```

Comment:
I don't understand this line.. you're listing all 4 possible positions anyway? :thinking: 
isn't this the same as `positions = ['top', 'bottom', 'left', 'right']`?